### PR TITLE
Plot tweaks

### PR DESCRIPTION
**Pull request by bwiernik**
Shorten some titles, format the VIF text more cleanly, use `plot.title.position = "plot"`

### CODE REVIEW

File: R/print.check_model.R [hunk 1]
```diff
@@ -141,8 +141,8 @@ print.see_check_model <- function(x,
 
   # make sure legend is properly sorted
   x$group <- factor(x$group, levels = c("low", "moderate", "high"))
-  levels(x$group) <- c("low (< 5)", "moderate (< 10)", "high (>= 10)")
-  names(colors) <- c("low (< 5)", "moderate (< 10)", "high (>= 10)")
+  levels(x$group) <- c("low (< 5)", "moderate (< 10)", "high (\u2265 10)")
```
Comment (IndrajeetPatil) [inline R/print.check_model.R:144]: Will this also work on Windows?

File: R/theme_abyss.R [hunk 2]
```diff
@@ -34,6 +35,7 @@ theme_abyss <-
       plot.title.size = plot.title.size,
       plot.title.face = plot.title.face,
       plot.title.space = plot.title.space,
+      plot.title.position = plot.title.position,
```
Comment (IndrajeetPatil) [inline R/theme_abyss.R:38]: Also need to add this to the .Rd files.

**Review by IndrajeetPatil — CHANGES_REQUESTED** [review]

File: R/print.check_model.R (hunk 1 above)
Comment (bwiernik) [reply R/print.check_model.R:144]: It does in my testing in RStudio view, R CMD graphics device, and knitting an Rmd document.

**Comment by IndrajeetPatil** [comment]
Looks good! Feel free to merge it.

**Comment by bwiernik** [comment]
Thanks! Can you update the tests that are now failing?

**Comment by IndrajeetPatil** [comment]
Do you know why there is no plot output here anymore?

https://easystats.github.io/see/articles/performance.html#overall-model-check-1

Reprex shows the same thing:

``` r
library(performance)
m <- lm(mpg ~ wt + cyl + gear + disp, data = mtcars)

check_model(m)
```

<sup>Created on 2021-06-29 by the [reprex package](https://reprex.tidyverse.org) (v2.0.0)</sup>

**Comment by IndrajeetPatil** [comment]
If I explicitly call `plot`, that doesn't work either.

``` r
library(performance)
m <- lm(mpg ~ wt + cyl + gear + disp, data = mtcars)

plot(check_model(m))
#> Error in xy.coords(x, y, xlabel, ylabel, log): 'x' is a list, but does not have components 'x' and 'y'
```

<sup>Created on 2021-06-29 by the [reprex package](https://reprex.tidyverse.org) (v2.0.0)</sup>

**Pull request merged**